downstream/modules/platform/ref-controller-automation-analytics.adoc

```diff
@@ -12,15 +12,7 @@ For opt-in of {Analytics} to have any effect, your instance of {ControllerName}
 As with Red Hat Insights, {Analytics} is built to collect the minimum amount of data needed.
 No credential secrets, personal data, automation variables, or task output is gathered.
 
-For more information, see xref:ref-controller-data-collection-details[Details of data collection].
-
-To enable this feature, turn on data collection for {Analytics} and enter your Red Hat customer credentials in the *Miscellaneous System settings* of the System configuration list of options in the {MenuAEAdminSettings} menu.
-
-image:configure-controller-system-misc-analytics.png[Miscellaneous System Settings]
-
-You can view the location to which the collection of insights data is uploaded in the *{Analytics} upload URL* field on the *Details* page.
-
-image:misc-system-details-analytics-url.png[Insights location]
+To enable this feature, see xref:proc-controller-configure-analytics[Configuring {Analytics}].
 
 By default, the data is collected every four hours.
 When you enable this feature, data is collected up to a month in arrears (or until the previous collection).
```

Comment:
You might want to highlight that this feature is automatically enabled by default.

Reply:
Updated.